cmd/tracee-rules/main.go

```diff
@@ -19,6 +19,10 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	signatureBufferFlag = "sig-buffer"
```

Comment:
nit: not really required - I would keep the current standard without defining a const var for it